apis/apps/v1beta4/emqxbroker_webhook_test.go

```diff
@@ -232,4 +240,23 @@ func TestBrokerValidateUpdate(t *testing.T) {
 		}
 		assert.Error(t, new.ValidateUpdate(old))
 	})
+
+	t.Run("valid emqxConfig can not update", func(t *testing.T) {
+		old := broker.DeepCopy()
+		new := broker.DeepCopy()
+
+		assert.Nil(t, new.ValidateUpdate(old))
+
+		new.Spec.Template.Spec.EmqxContainer.EmqxConfig["name"] = "emqx-test"
+		assert.Error(t, new.ValidateUpdate(old))
+
+		new.Spec.Template.Spec.EmqxContainer.EmqxConfig["name"] = "emqx"
+		new.Spec.Template.Spec.EmqxContainer.EmqxConfig["cluster.dns.app"] = "emqx-test"
+		assert.Error(t, new.ValidateUpdate(old))
+
+		new.Spec.Template.Spec.EmqxContainer.EmqxConfig["name"] = "emqx"
+		new.Spec.Template.Spec.EmqxContainer.EmqxConfig["cluster.dns.app"] = "emqx"
+		new.Spec.Template.Spec.EmqxContainer.EmqxConfig["listener.tcp.internal"] = "0.0.0.0:1884"
+		assert.Nil(t, new.ValidateUpdate(old))
```

Comment:
Check if the new configuration can be updated successfully if it does not contain name and cluster, just add new filed or just change `listener.tcp.internal`

Reply:
> Check if the new configuration can be updated successfully if it does not contain name and cluster, just add new filed or just change `listener.tcp.internal`

A test case for the above has been added